src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
```

Comment:
Classes must have access modifiers; otherwise, the default will be used, which isn't a good practice. Consider adding an access modifier to the Lottery class.